src/ImageSharp/Formats/Tiff/TiffDecoderCore.cs

```diff
@@ -775,8 +791,7 @@ private int CalculateStripBufferSize(int width, int height, int plane = -1)
     /// <returns>The image width.</returns>
     private static int GetImageWidth(ExifProfile exifProfile)
     {
-        IExifValue<Number> width = exifProfile.GetValue(ExifTag.ImageWidth);
-        if (width == null)
+        if (!exifProfile.TryGetValue(ExifTag.ImageWidth, out IExifValue<Number> width))
         {
             TiffThrowHelper.ThrowImageFormatException("The TIFF image frame is missing the ImageWidth");
```

Comment:
Just realized these are not using the correct exception type. It should be `InvalidImageContentException`